src/components/ebay-toggle-button-group/examples/columns.marko

```diff
@@ -0,0 +1,26 @@
+<ebay-toggle-button-group columnsSM=3 columnsXS=2 columnsMD=6 columnsXL=8 ...input>
```

Comment:
Do we need `columns` and `controlled`? Aren't they the same thing?

Maybe we can combine them into a single one, titled "Exception Cases" or "Preferred Columns?"

Reply:
Controlled is an important code example for devs who want to hold state in the parent instead of letting our component do it, but it may not be necessary to have in storybook

Reply:
Will leave it. There's no other way for devs to know how to use columns